lib/gollum/views/helpers.rb

```diff
@@ -0,0 +1,35 @@
+require 'yaml'
+
+module Precious
+  module Views
+    module SprocketsHelpers
```

Comment:
To clarify further, using the Sprockets helpers is necessary because Sprockets generates assets with an md5-fingerprint. So the correct parth to the application stylesheet will e.g. be `assets/app-580166b40601dad84878e5922b249013547baf5078617b192d9a4eb8d4e32fab.css`. The `stylesheet_tag` helper and its family is necessary to generate a link to the asset that uses the correct fingerprint.